go.mod

```diff
@@ -3,21 +3,30 @@ module github.com/replicatedhq/embedded-cluster-operator
 go 1.21.0
 
 require (
+	github.com/ghodss/yaml v1.0.0
+	github.com/k0sproject/dig v0.2.0
 	github.com/k0sproject/k0s v1.28.5-0.20231116142149-82f76181191c
 	github.com/k0sproject/version v0.4.2
+	github.com/ohler55/ojg v1.21.0
 	github.com/onsi/ginkgo/v2 v2.13.2
 	github.com/onsi/gomega v1.30.0
 	k8s.io/api v0.29.0
 	k8s.io/apimachinery v0.29.0
 	k8s.io/client-go v0.29.0
 	sigs.k8s.io/controller-runtime v0.16.3
+	sigs.k8s.io/yaml v1.3.0
```

Comment:
do you actually want two yaml libraries here?

Reply:
as far as I understand `fmtconvert "github.com/ghodss/yaml"` is used specifically for it's ability to move data between yaml and json format. 

`"sigs.k8s.io/yaml"` is in use because it's able to correctly marshal/unmarshal kubernetes-like objects

Reply:
for reference, we do the same here: https://github.com/replicatedhq/embedded-cluster/blob/main/pkg/config/config.go

Reply:
actually yeh, you're right. sigs.k8s.io/yaml already provides a YAMLtoJSON function 🤦